draft-ietf-quic-recovery.md

```diff
@@ -486,6 +486,12 @@ Using max(SRTT, latest_RTT) protects from the two following cases:
 * the latest RTT sample is higher than the SRTT, perhaps due to a sustained
   increase in the actual RTT, but the smoothed SRTT has not yet caught up.
 
+An endpoint might consistently record RTT samples as 0 in extremely low latency
+networks, leading to a smoothed_rtt of 0.  Consequently, the endpoint could
+declare all packets as lost immediately upon sending them.  To avoid declaring
```

Comment:
This part isn't true.  There would still have to be reordering, since you need a later sent packet to be ACKed.

Reply:
Good point. I've changed the text now to say that srtt of 0 means there's no reordering tolerance.